examples/cfd/07_cavity_flow.ipynb

```diff
@@ -0,0 +1,566 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">&lt;ipython-input-12-e539ef5b065a&gt;:34: RuntimeWarning: invalid value encountered in true_divide
  l1norm = (np.sum(np.abs(_p.data[:]) - np.abs(_pn.data[:])) /
</pre><p>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1368/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
The initial solution to the pressures has everything as zero. This means you divide by zero on the first time step. 
This can be avoided either by skipping the pressure solve on the first step, or adding epsilon to the denominator.